README.md

```diff
@@ -117,20 +117,39 @@ To execute cucumber test scenarios
 bundle exec cucumber
 ```
 
-## Developing Cucumber tests
+## Testing external services
 
-Some cucumber feature test use VCR to record/store mock results. This is currently required
-because of internal API calls (calling our own API) for certain endpoints (case worker claims
-in particular). To create a new feature/scenario that relies on such endpoints you will therefore
-need to record a new "cassette", as below.
 
-#### Create new VCR cassette
-##### (for cucumber features that call the internal API or a microservice)
+#### LAA fee calculator API
+##### RSpec
+Some rspec unit tests require VCR cassettes for the LAA fee calculator API external service. These specs are tagged with `:fee_calc_vcr` so can be targetted using rspec cmdline options.
+
+```bash
+# run specs requiring LAA fee calculator API call stubs/cassettes
+$ rspec --tag fee_calc_vcr
+
+```
+
+Changes to the calling of the LAA fee calculator API will most likely require you to rerecord the VCR cassetes that stub these calls. To rerecord
+VCR cassettes you can delete the existing ones (in `vcr/cassettes/spec`). They will be recreated when the specs are run.
+
+It is a good idea to do this when changes occur to the LAA fee calculato API too.
+
+##### Cucumber
+Some cucumber features require VCR cassettes to stub calls to the LAA fee caclculator API. These features require and are tagged with a `@fee_calc_vcr` tag. To re-record the cassettes delete the existing ones and run the feature again. See [Create a new VCR cassette](#new_cassette) below for more.
+
+#### Internal API
+Some cucumber feature tests use VCR to record/store mock results the internal API calls (calling our own API) for certain endpoints (case worker claims in particular).
+To create a new feature/scenario that relies on such endpoints you will therefore need to record a new "cassette", as below.
+
+<a id='new_cassette'></a>
```

Comment:
not needed for hyperlinking... see separate comment

Reply:
can't get this working in my local markdown viewer (perhaps it is github specific markdown??) or i have it wrong:

heading:
```
##### Create new VCR cassette
```

link: 
```
[Create a new VCR cassette](#create-new-vcr-cassette)
```